services/data_deletion/clean_deleted_data_from_db.py

```diff
@@ -0,0 +1,23 @@
+from helpers.database import database_uri
+from sqlalchemy import create_engine, MetaData
+
+
+class DataDeletion:
+    def clean_deleted_data(self):
+        """
+        This method deletes data that has been marked for
+        deletion from the database
+        """
+        print("cleaning data...")
+        database_engine = create_engine(database_uri)
+        metadata = MetaData()
+        metadata.reflect(bind=database_engine)
+
+        for table in reversed(metadata.sorted_tables):
+            # for each table delete all data marked for deletion
```

Comment:
KIndly include your comments immediately after the function definition. Use docstrings https://www.python.org/dev/peps/pep-0008/#block-comments